packages/fluentui/react-builder/src/components/SRMC-Definitions.tsx

```diff
@@ -0,0 +1,222 @@
+const SRMC: { [key: string]: any } = {
```

Comment:
you since this is a object of constants, you shouldn't really define any type for this. Let typescript inference do its job :)

```suggestion
const SRMC = {
```

Reply:
When I apply your suggested change, that is, remove the `{ [key: string]: any }` typ, I get the following error:

```
TypeScript error in C:/Users/asamec/Dropbox/Work/Microsoft/Programming/sr-message-computation-ts/Windows/sr-message-computation/src/MessageComputer.tsx(47,27):
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ 'Win/JAWS': { '[default]': string[]; 'input:text':
string[]; 'role=combobox': string; a: string[]; 'role=menuitem': string[]; 'role=menuitemcheckbox': string; 'role=menuitemradio': string; select: string;
'role=tab': string; 'role=option': string; }; }'.
  No index signature with a parameter of type 'string' was found on type '{ 'Win/JAWS': { '[default]': string[]; 'input:text': string[]; 'role=combobox':
string; a: string[]; 'role=menuitem': string[]; 'role=menuitemcheckbox': string; 'role=menuitemradio': string; select: string; 'role=tab': string; 'role=option':
string; }; }'.  TS7053

    45 |         if (definitionType === 'readingOrder') {
    46 |             // Begin if 1
  > 47 |             definitions = SRMC.readingOrder[platform];
       |                           ^
    48 |         } else if (definitionType === 'stateRules') {
    49 |             // else if 1
    50 |             definitions = SRMC.stateRules[platform];
This error occurred during the build time and cannot be dismissed.
```